data/statuses.js

```diff
@@ -462,6 +462,31 @@ let BattleStatuses = {
 			return this.chainModify([0x14CD, 0x1000]);
 		},
 	},
+	smarttarget: {
+		name: 'smarttarget',
+		id: 'smarttarget',
+		num: 0,
+		duration: 1,
+		onRedirectTargetPriority: -1,
+		onRedirectTarget(target, source, source2, move) {
```

Comment:
I'm actually really confused how this handler isn't applying for redirected moves.